enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -294,11 +301,16 @@ msgid "Username"
 msgstr ""
 
 #: enterprise/templates/enterprise/admin/manage_learners.html:99
-#: enterprise/templates/enterprise/admin/manage_learners.html:124
+#: enterprise/templates/enterprise/admin/manage_learners.html:126
 msgid "Linked Date"
 msgstr ""
 
-#: enterprise/templates/enterprise/admin/manage_learners.html:144
+#: enterprise/templates/enterprise/admin/manage_learners.html:100
+#: enterprise/templates/enterprise/admin/manage_learners.html:127
+msgid "Enroll"
+msgstr ""
+
+#: enterprise/templates/enterprise/admin/manage_learners.html:148
 msgid "Link learners"
 msgstr ""
 
```

Comment:
Since I can't comment directly on lines 320-362, I've added comments for these lines below.

**ll. 319-320**

To look up third party authentication providers, you must install this package in an edX environment.

**l. 324**
The file must be an .xls file.

**l. 328**
OK

**l. 333**
OpenEdX --> Open edX

**ll. 339-340**
OK

**ll. 346-347**
get --> receive, strike "any"

**ll. 353-354**
{provider} requires data sharing consent. If you do not agree to share your data, you will be redirected to the sign in page.

**ll. 360-361**
{provider} requires data sharing consent. If you do not agree to share your data, you cannot receive discounts from {provider}.